treeherder/perf/migrations/0021_remove_application_from_constraint.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.6 on 2019-11-11 16:25
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0015_add_repository_tc_root_url'),
+        ('perf', '0020_add_application_field'),
```

Comment:
I am not sure what this `perf` reference is, but looking at `0020_add_application_field`: this migration appears to undo the indexing